time stamp

ros_rl/ThingMsg obs

float64 reward

bool terminal


